bazel/repositories.bzl

```diff
@@ -896,24 +895,19 @@ def _com_github_wamr():
 def _com_github_wasmtime():
     external_http_archive(
         name = "com_github_wasmtime",
-        build_file = "@envoy//bazel/external:wasmtime.BUILD",
+        build_file = "@proxy_wasm_cpp_host//:bazel/external/wasmtime.BUILD",
```

Comment:
It makes sense to me to switch to depending on proxy-wasm-cpp-host's BUILD file for Wasmtime, so that we only need to maintain it in one place. Just to confirm, @phlax is it ok for Envoy to lean on one external dependency's build file for another external dependency, or does Envoy aim to provide all supplementary BUILD rules itself?

Reply:
i dont think we have any policy on this and i think just reusing the existing BUILD file reduces duplication/maintenance - i think we only use this dep in the context of cpp-host anyway